### PR TITLE
Add ARM support

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments

Adds ARM support

## Testing

If you have an Intel-based macOS system, make sure you have `virtualbox` installed, and then you should be able to run `./start.sh --provider=virtualbox` and it will spin up an Ubuntu 20.04 VM for you.

If you have an ARM-based macOS system, follow the `Getting Started` in the README, and you should also get an Ubuntu 20.04 VM available after provisioning.

## Pull Request Checklist

Thank you for submitting a contribution to the `ubuntu-sandbox`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:

- [x] Rebase your branch against the latest commit of the target branch
- [x] If you are adding a dependency, please explain how it was chosen
- [x] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results
- [x] If there is an issue associated with your Pull Request, align your PR
  title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  systemctl restart docker
+  usermod -aG docker vagrant
+fi
+
+# Firewall setup
+ufw default allow outgoing
+ufw default deny incoming
+ufw allow in on eth0 to any port 22 proto tcp
+ufw logging medium
+echo "y" | ufw enable
```
Comment (digeseiso) [inline init.sh:49]: @JonZeolla have you considered using `printf` instead of echo? the former is more consistent across platforms

**Review by digeseiso — APPROVED** [review]

File: init.sh (hunk 1 above)
Comment (JonZeolla) [reply init.sh:49]: Can you give an example of where this may be inconsistent?

Comment (digeseiso) [reply init.sh:49]: @JonZeolla Please review https://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo it explains it better than i could here `printf`

Comment (JonZeolla) [reply init.sh:49]: Ah okay cool.  Learn something new today. It seems like for our case echo is just fine, but when you start getting into the echo -e world (like I have in other places) we should really be replacing that with printf.  Good to know!  Thanks @digeseiso

**Pull request merged**